scripts/updateMainReference.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
What is this file for ?

Reply:
Please add this kind of utility file in a dedicated commit, with the associated doc and comments

Reply:
I meant a dedicated PR, sorry